3-enrich/scala-common-enrich/project/Dependencies.scala

```diff
@@ -103,6 +106,7 @@ object Dependencies {
     val scalaWeather     = "com.snowplowanalytics"      %% "scala-weather"                 % V.scalaWeather
     val scalaj           = "org.scalaj"                 %% "scalaj-http"                   % V.scalaj
     val gatlingJsonpath  = "io.gatling"                 %% "jsonpath"                      % V.gatlingJsonpath
+    val sttp             = "com.softwaremill.sttp"      %% "core"                          % V.sttp
```

Comment:
we already have scalaj on the classpath, I'd be in favour of using it instead of introducing a new dependency.

Reply:
removed it, but had to add timeout to HttpClient class